mkdocs.yml

```diff
@@ -162,7 +162,8 @@ nav:
           - Crab Chain: "evm/chains/crab.md"
           - Pangolin Chain: "evm/chains/pangolin.md"
           - Rollup Testnet: "evm/chains/rollup.md"
-      - Available Wallets: "evm/wallets.md"
+      - Available Wallets:
+          - Overview: "evm/wallets.md"
```

Comment:
The other changes LGTM. Let's keep the outline category simple to view. Suggest adding Overview page for the topics with more than one page, as for the Governance and Available Wallet. No need to do this now, add it when necessary later.

Reply:
I just feel it belong to `Chains and Exploreres` with a wrong style.